src/Controller/RequestController.php

```diff
@@ -114,7 +114,7 @@ public function handleBatchRequest(Request $request, array $queries = []) {
     // Walk over all queries and issue a sub-request for each.
     $responses = array_map(function ($query) use ($request) {
       $method = $request->getMethod();
-      $parameters = $method === 'GET' ? $query : [];
+      $parameters = $method === 'GET' ? $query : $request->query->all();
       $content = $method === 'POST' ? json_encode($query) : '';
```

Comment:
What you say it is almost the same as I want. Here is my idea:
- if it is a GET request, I want $request->query->all() EXCEPT for "queries" (as you said).
- if it is a POST, I want the entire POST without 'queries' in the $content variable, AND the entire GET in the $parameters.
My use case was that the initial call to graphql was a POST to something like /graphql?domainId=somedomain, but the subrequest ignored the ?domainId=somedomain, while forwarding the POST body/content.

I see that the current change I did does not really cover these cases, so if what I described above makes sense, I can update the implementation.